### ISSUE TITLE
labels is not appear when StackedBarSeries is not visible and animationDuration is not zero

### ISSUE THREAD

**Issue by robertoltrocha** [syncfusion/flutter-widgets#74]
Hi,
When we set the visible mode to false, the label for this StackedBarSeries does not appear when we use animationDuration > 0.

  ```
  StackedBarSeries<SalesData, String>(
                        isVisible: false,
                        animationDuration: 2000,
                        enableTooltip: true,
                        name: "Geral",
                        dataSource: getColumnData(),
                        xValueMapper: (SalesData sales, _) => ConfigStrings.dateMMyyyy.format(sales.data),
                        yValueMapper: (SalesData sales, _) => (sales.percentual),
                        dataLabelSettings: DataLabelSettings(
                          isVisible: true,
                        )),
```

![WhatsApp Image 2020-12-03 at 21 46 03](https://user-images.githubusercontent.com/30963205/101107826-edcc3a00-35b1-11eb-82cb-e10953be04cc.jpeg)
![WhatsApp Image 2020-12-03 at 21 46 40](https://user-images.githubusercontent.com/30963205/101107831-f15fc100-35b1-11eb-88e1-d28ad008f7c0.jpeg)

show labels only if animationDuration is zero when isvisible is false.
**isVisible: false,
animationDuration: 0,**, I don't know if this is expected.
![WhatsApp Image 2020-12-03 at 21 48 12](https://user-images.githubusercontent.com/30963205/101107851-fe7cb000-35b1-11eb-9dc9-db817706a940.jpeg)

**Comment by SriramKiranSenthilkumar** [comment]
Hi @robertoltrocha,

Greetings from Syncfusion. We have analyzed your scenario with the provided information and we are able to replicate the reported issue regarding the data labels are not rendering for the series when its isVisible set as false and animationDuration > 0 in the chart. We will consider this as a bug and will include the fix for the reported issue in our upcoming weekly patch release which is expected to be rolled out by the middle of this month. We will update you once the release is rolled out and we appreciate your patience until then.

Regards,
Sriram Kiran

**Comment by Dharanipalani** [comment]
Hi @robertoltrocha, 
  
The issue got resolved in the main release(v18.4.30). So, kindly update our flutter charts to v18.4.30 or above to resolve the reported scenario. The latest version can be downloaded from below. 
  
Pub: https://pub.dev/packages/syncfusion_flutter_charts/versions/18.4.32+1 
 
 
Thanks, 
Dharanitharan. P
